software/litepcie-kernel-module/config.h

```diff
@@ -55,7 +55,7 @@
 #define DMA_CHANNEL_COUNT DMA_CHANNELS
 #define DMA_BUFFER_PER_IRQ 32
 #define DMA_BUFFER_COUNT 256
-#define DMA_BUFFER_SIZE 8192
+#define DMA_BUFFER_SIZE 65536
```

Comment:
@maleadt this seems okay to do, correct?
This puts it right on the page limit for the GPU, but it doesn't seem to be a problem.

Reply:
It's been a while since I wrote the page handling logic, but I think this is OK.